### DISCUSSION TITLE
Exporting and Soundfonts/MIDI Library

### DISCUSSION THREAD

**Discussion by Nathul123** [helge17/tuxguitar#425] [General]
I've found that when I export WAV audio, Tux doesn't use my selected MIDI library (Apollo GMGS), and instead defaults to the base Tux MIDI library. Is there something I'm missing here or is this not a feature yet?

**Comment by blindfaced** [comment]
i've got scales enlarged 
[scales.txt](https://github.com/user-attachments/files/15896787/scales.txt)
if wanted

**Reply by Nathul123** [reply]
I think you responded to the wrong thread, bud.

**Comment by guiv42** [comment]
> Is there something I'm missing here

I don't think so.
I suggest creating a new issue. Please take care to precise the version of TuxGuitar you are using, the version of operating system, and most importantly the detailed sequence to reproduce. Especially: step-by-step description of how you configure TuxGuitar to use a specific soundfound (not 100% sure but I think there are several ways to do that)

**Comment by guiv42** [comment]
tracked by #730
